src/Evolution/Systems/RadiationTransport/M1Grey/Sources.cpp

```diff
@@ -0,0 +1,83 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/RadiationTransport/M1Grey/Sources.hpp"
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace {
+struct TildeSUp {
```

Comment:
So, it may be worth discussing where to put this. For the moment, we have a TildeS tag templated on frame in the ValenciaDivClean tags, and a TildeS tag templated on frame AND neutrino species in the M1Grey tags. 
This tag doesn't know about the neutrino species, so conceptually it is closer to the MHD tag... and the MHD equations may need TildeSUp as well. So it's not clear to me that the M1Grey tags are where it should go.

Also, given that TildeSUp works, i assume that it's ok for a tag definition to be overloaded (in the sense that 2 definitions are templated on different classes)?

Reply:
I believe tags making reference to a neutrino species should remain (for now) in the corresponding radiative transport namespace, regardless of whether they share the same name as the hydro tags.

I am not sure if, after an additional cleanup in ValenciaDivClean (or even in RelativisticEuler), TildeSUp will be useful there. For now, I'd stick this TildeSUp to the M1Grey tags (provided you will need this in other functions too). Maybe in the future, with the addition of things like viscosity, we will end up factoring out some of these definitions.

Something related: I'm not sure if  `Up` should be the nomination here. For vectors whose "down" forms are needed as arguments, we usually say `OneForm`. If you're working under the assumption that the 1-form of tilde S is "the" tilde S (as it is in GR(M)HD), `TildeSVector` might be more appropiate.

Reply:
Ok, will move that into the neutrino tags for now, we can revisit later.